netplugin/plugin/netplugin.go

```diff
@@ -241,6 +288,7 @@ func (p *NetPlugin) Reinit(cfg Config) {
 	}
 
 	defer func() {
```

Comment:
agreed. It doesnt stack up the defers. Looks like we might have bug also in the previous code. will fix it.